harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -901,7 +908,7 @@ def get_trial_list_after_successful_run(
 
     def should_stop_lineage(self, trial: DSATTrial) -> bool:
         # General conditions
-        failed_on_min_mbs = trial.error and trial.mbs <= trial.search_data.lo
+        failed_on_min_mbs = trial.error and trial.search_data and trial.mbs <= trial.search_data.lo
```

Comment:
Similar to the above: not sure when `trial.search_data` would happen? Seems like there should be an `assert trial.search_data`, if anything.

Reply:
I can do that. assert makes more sense in this case